cataclysmbnteam/Cataclysm-BN
# Update UndeadPeople tileset

Update DeadPeople tileset



#### Summary
SUMMARY: Content  "Update DeadPeople tileset"


#### Purpose of change
Update DeadpPeople tileset


#### Describe the solution
Update tileset directory.



#### Describe alternatives you've considered
None


#### Testing
1) Make sure that DeadPeople tileset selected.
2) Load game or start new one.
2) Game loads correctly and there are no tileset related errors.


#### Additional context
